bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -201,6 +258,18 @@ impl Vm {
             registered: true,
         }
     }
+
+    /// Get the store context used in the WasmEdge_VMContext.
+    pub fn get_store(&self) -> Option<Store> {
+        let store_ctx = unsafe { wasmedge::WasmEdge_VMGetStoreContext(self.ctx) };
```

Comment:
A straight idea to support the syntax `vm.store.get()` is to add `pub store: &Store` in the declaration part of `struct Vm`, and then add `pub get(&self)` interface in `impl Store`. That should work. The syntax looks good, but I think that could make the design a little bit complex. I suggest the -sys layer keep as simple and straightforward as possible so that the future high-level API design could have enough space to abstract.

Reply:
It may be good here if there is no cost on the abstraction, and we can implement it in the same crate.  Too many high levels api, and doing a little thing in each will confuse the user.  If it is impossible, let's keep it simple as current implementation.  Thanks.